### DISCUSSION TITLE
I am so frustrated!

### DISCUSSION THREAD

**Discussion by hanneslsm** [WordPress/gutenberg#50129] [General]
I know, the title is a little clickbaity, but hear me out.

Let me start by saying that this is not another typical 1-star review calling for a return to the classic editor. In fact, I wholeheartedly believe in the vision behind Gutenberg and the power of open source. As a web designer, I've been following the development of Gutenberg closely since years, from the official updates to GitHub issues and PRs. I admire the team's hard work and dedication to making it better. I'm particularly impressed by how @Thelmachido handles negative reviews in the support forum with grace, @jameskoster's stunning mockups, @jasmussen's deep knowledge of the whole WordPress system, @annezazu's friendly nature that keeps the whole crew together or @richtabor, who seems to prioritise a holistic and easy-to-use product, just to name a few I regularly see here. There are also many more invaluable contributions from other essential team members.

However, I can't deny the deep frustration I feel. I love WordPress and don't want to leave it behind, but the current state of Gutenberg's user experience is driving me to consider alternatives like Webflow, Wix, and Shopify, which I never thought I'd do. It feels like I'm losing the battle for an open web, and it goes against my conviction.

Recently, I had a client who needed an online shop. I struggled with choosing WooCommerce & Gutenberg because Gutenberg still feels immature. Reluctantly, I had to propose using Shopify, which felt like admitting defeat.

Last year, I decided to take the leap and quit my job to become a full-time web designer. Throughout this journey, I've been eagerly waiting for Gutenberg to reach its full potential. I've been patient, I've been supportive, and I've been hopeful. But, unfortunately, I've also been frustrated.
You see, as much as I love Gutenberg and the idea of open source, I've had to rely on apps like Elementor to make sure my clients can easily navigate their websites. I've struggled with the idea of recommending Gutenberg to others, fearing it could harm my reputation as a web designer. Despite countless updates, I still see unresolved issues that have been around for years.
It's difficult for me to rely on several third-party plugins to extend WordPress and Gutenberg’s capabilities like Elementor, only because Gutenberg still has so many unnecessary flaws. The different UX behaviours that the plugins introduce, security risks, and performance issues only raise more questions (and costs) for my clients. Many plugins have their justification and i love the concept that Wordpress is extensible through plugins. But when it comes to basic features that need to be filled with plugins, it just doesn't feel good.

My frustration grows when I try to contribute and raise issues with concrete examples of improvements, only to have my suggestions dismissed, and tickets closed because of duplicates that somehow have already disappeared into oblivion. These weren’t fancy special edge cases, but very basic features. I can't help but feel ignored and disregarded, and it's exhausting. There is so much going on in the development of Gutenberg, and I love it! However, I think it's crucial to focus at least for a few weeks on very basic UX improvements.

Let me give me some examples of what I mean: adding basic dimension controls for the video block (#50018), addressing gallery block issues (#50022 and #50021), or improving the UX when changing preferences (#50017). These low-hanging fruits can significantly improve the user experience without losing sight of Gutenberg's long-term goals. I recently was in a client's call and I was seriously ashamed of how often I had to say “this cannot be currently done, I'll install a plugin" or "you have to do it like this…" only to look at confused faces afterwards.

If WordPress 6.3 doesn't address these basic UX issues, I fear I'll have to move away from it entirely, and I don't know when or if my trust can be restored. It's a painful thought, especially since I love this community, believe in the mission of WordPress like democratising publishing (which doesn’t feel like when you have to invest a lot of time to get basic things working or invest money for plugins) and in the power of open source.

I wish I could get more involved in development, but I have to look after my own business. I would even be open to helping (if there is a sponsor out there, let me know) - at least until the very basic features are fixed. However, if the whole thing when trying to contribute is so frustrating, I don't even know if it's worth it.

I really think that creating a user-friendly and accessible Gutenberg experience should be prioritised more. Basic UX issues should be addressed to ensure that the mission and values of WordPress remain intact and continue to drive the platform's success. I know that you have heard this a lot already. But if this does not change, I see the fight for a free internet as lost. This is what frustrates me the most.

**Comment by jasmussen** [comment]
I share this frustration, I even have my own set of top items I want fixed (#49501 comes to mind just to mention one). I want the same, for 6.3 to be a very polished release after a longer period of completing features. I hope we can get there!

**Reply by hanneslsm** [reply]
Thank you for your response. It feels good to know that even main contributors like you share these frustrations, and that gives me hope and motivation to keep helping. Your example is also very good – it shows what could be improved quickly.

I think the main issue causing this frustration is that there's no solid prioritization process or tracking system for these kinds of essential UX issues. It seems like a lot of the current progress is more focused on long-term goals or specific feature details.
I guess a good way to start would be creating a simple user journey that outlines all the steps that are made and all the obstacles and bugs that are encountered. I'm willing to help create these issues. However, as I stated above, creating the issues seems to be not enough because they might not be touched for years.

Probably we would also need to create a tracking issue that bundles all the issues, then? What else? How does the process work within the Gutenberg team (and Automattic?) when tackling such a project? I don't want other goals to be neglected in the process.

I also think a developer that is dedicated to addressing this project would be very beneficial. I love, for example, how @andrewserong is currently focusing on the List View, and the rapid progress being made there.
Is there a process for getting a dedicated team for this kind of work? I fear that even when creating all the issues and the tracking ticket, everything then quickly comes to nothing (or at least the progress is not as fast as needed).

**Reply by hanneslsm** [reply]
If anyone wonders what I mean with starting by creating a simple user journey where the issues could be derived out of, here is an example: https://github.com/WordPress/gutenberg/issues/50021#issuecomment-1527711429

**Comment by annezazu** [comment]
Thanks so much for taking the time to share your feedback and frustrations @hanneslsm. I don't take it for granted when it's often easier in this world to disengage rather than to take the time to engage. I use WordPress everyday, including on my personal site where I've been running a block theme for a long while now, and totally hear you on many of the pain points you expressed. Running the FSE Outreach Program also brings me into regularly contact with various pain points! 

>I really think that creating a user-friendly and accessible Gutenberg experience should be prioritised more. Basic UX issues should be addressed to ensure that the mission and values of WordPress remain intact and continue to drive the platform's success. 

Part of what both makes me optimistic and drives me to improve what we're creating hinges on the fact that we're ultimately bringing blocks everywhere. By doing so, any improvements to blocks (from better transform options to more tooling) has a cascading impact everywhere that block is used, from the post editor to potentially the site editor. The same is true for improving things like drag & drop or list view. We have a lot to work on though as you state and I want to encourage folks to continue sharing pain points. 

In terms of adding more polish, I want to note a new board was opened to gather issues: https://github.com/orgs/WordPress/projects/96/views/1 This won't cover everything inherently. In general, the aim is to have items on here that are quickly actionable for developers that don't add any new features and don't require additional design thought or help. This is a narrow scope but also it's yet another starting point and I'm optimistic we can continue to improve WordPress thanks to the power of relentless iteration.

**Reply by hanneslsm** [reply]
Thank you for your kind reply! I am sure you are aware of many issues with the outreach program, and I have great respect for how enduring and optimistic you remain.

Bringing blocks everywhere is a goal I fully support, and when we get there, it will be an amazing product. However, it's also important to focus on the current experience and take the time to evaluate if it's still easy to use. Otherwise, as mentioned above, Gutenberg will be good in a few years, but many users will have turned away because other services currently deliver better experiences, aka a better product.

> In terms of adding more polish, I want to note a new board was opened to gather issues: https://github.com/orgs/WordPress/projects/96/views/1 This won't cover everything inherently. In general, the aim is to have items on here that are quickly actionable for developers that don't add any new features and don't require additional design thought or help.

The new board makes me enormously happy! In fact, raising awareness and prioritizing these kinds of issues were the main goals of this post. We're all aware of the bigger features we want to see implemented, but focusing on "smaller" issues is just as important. Thank you! 
Can you provide more information on how this board was created? What's the process for deciding what will be on the board and what will not? Who makes the decisions and when? I can think of some more issues that haven't had a GitHub issue created for them and would fit on the board.

**Reply by annezazu** [reply]
> Can you provide more information on how this board was created? What's the process for deciding what will be on the board and what will not? Who makes the decisions and when? I can think of some more issues that haven't had a GitHub issue created for them and would fit on the board.

The board was created by folks involved in WordPress releases and project leadership (Matías), including folks you mentioned on this thread (Joen, Rich, myself). It's going to be curated on an ongoing basis by those same folks. It was created ahead of 6.3 to help provide more polish, as the name suggests, and it's intended to be for developers looking for quick items to tackle that make a meaningful difference to the experience but that don't require design assistance. This stands in contrast to big projects tracked here that require a ton of collaboration between design and development: https://github.com/WordPress/gutenberg/issues/33094

Feel free to drop issues you'd like to see considered in this thread and I'll see what can be added based on that criteria.

**Reply by cbirdsong** [reply]
I saw #50021 was added to the board and then later removed - why is that?

**Reply by richtabor** [reply]
The Polish board is for ready-to-pick-up actionable issues. This issue needs follow-up design work—perhaps a prototype—to prove out the concept, or suggest an alternative, before it gets picked up.

**Reply by annezazu** [reply]
Exactly that ^ I expect to see that happen a bit in the next few weeks as issues are triaged and evaluated before the right balance is found.

**Comment by richtabor** [comment]
> Thanks so much for taking the time to share your feedback and frustrations @hanneslsm. I don't take it for granted when it's often easier in this world to disengage rather than to take the time to engage. 

Very well said. 

@hanneslsm Good experience (especially at this scale) takes time and thoughtful execution. Every well-formed issue is appreciated, but creating that issue doesn't mean it will be committed to in the same light. It does however mean we can share and discuss what's important to each of us. Bringing up those pain points helps us empathize better with all the ways people use WordPress. 

Passion fuels progress, and we need more of that—so thank you. :)

**Reply by hanneslsm** [reply]
> but creating that issue doesn't mean it will be committed to in the same light.

Absolutely! I fully agree on with you and I have respect for the way the Gutenberg team thinks in the bigger picture and just doesn't jump on the bandwagon and only addresses the problems of those who shout the loudest (intrinsic design is a great example). 
Still, there are things that would bring consistency to the editor, improve the experience, could be changed easily and fast and without sacrificing the bigger goals. The new UX & Polish board is a great start!

**Comment by Stephan256** [comment]
### Is this the beginning of the end for WordPress Mister Mullenweg?

**I share the frustration!**

WordPress is a wonderful tool, its success is proof of that.

I have been developing with since version 2.7, so it has been part of my daily life for more than ten years.
Thank you Mister Mullenweg! Thanks to all contributors.

### But what happens now?

The transition to the full site editor is an interesting and exciting concept.
I have been working with the FSE for over a year and I note with bitterness that it is still not viable.
I don't count the problems, the inconsistencies, the blocks that just lack two options to be complete.
I waited for the end of the beta version and this one still does not seem successful to me.
**Please finalize this tool before embarking on other projects or unnecessary features!**
_I am sorry to say this but it is too long and incomplete!_

**I don't want to use Elementor, yet have you tried and figured out why so many designers are using it? It is complete !**

### But what about the Gutemberg and FSE documentation?

To start, you choose to do full js and React JS is yours and I respect it. But you have put a lot of developers on the floor, without offering a gateway from PHP.
Yes I manage to make extensions for the editor block but what a hassle to achieve it.
The second great strength of WordPress has been its complete and up-to-date documentation.
The last time I took over the documentation to manage the blocks, it had become obsolete and no longer worked!
Why do you have to glean information from several sites? wordpress.org and fullsiteediting.com?
Yes I know it's not easy to manage everything, to be perfectly consistent and up to date.
But please let's stay serious and professional so that WordPress lives for a long time!
I want it as much as you.

**We no longer want to hear: "Drupal does this natively" or "With Elementor it's easy"**

### We need a Wordpress PRO!


**WP++: Here are some jumbled proposals for management on the backoffice side:**

When we have a lot of taxonomies with a lot of terms, the page editor interface is not really workable.
Why not use all the available space in a new tab overlaying the article content? Please.
In the list of posts It's time to review the interface to make it more efficient: the filters take up too much space and could be more explicit.

We used to have Widgets, easily programmable in PHP, can you redo these for editor blocks?
A block with the options in the sidebar and a preview queried in REST API, to be able to manage almost everything in PHP.
This would be a great help for the developers and feedback for many of us.


### Let's finish the site editor and further improve Gutenberg:

Can we add the notion of group for templates and template-parts?
Indeed I have a lot of custom post-type and a lot of taxonomies, if to that I add derivatives and management by language it becomes unmanageable. You have to think of us and be able to add folders and sub-folders in the site editor interface and in the template folder structure: \templates\books\*, \templates-parts\books\* or \templates-parts \header\*

As a designer / developer, managing templates and template-parts, between the BDD version and the version between the parent template and the child template is tedious.
The Create Blocks Themes plugin is hopelessly buggy (except for the "Twenty Twenty-Three" theme of course)
So I myself created an interface to view at a glance what is "customized" or not and in one button I save these in the corresponding file of the right child / parent theme then I delete the customization.

The possibility of adding display options on the pages/cpt in the editor. (via programming)
For example to have the following checkboxes: Show title, Make full width, No Top Margin, Show breadcrumbs, Alternate background
The page templates cannot respond to all the possible combinations.


### The blocks:

To start: why when you want to paste (or move) a block in a column, a group etc. do you have to create a paragraph?

A little more wysiwyg, especially for some margin settings, and you'll be fine.

Border, radius Ok and Box Shadow please, why stop there?

**Button**: Why can we modify the border radius and not the border? It drives me crazy and my clients too!

### QueryLoop:
- Taxonomies:
- where is the option to include all taxonomies and not just one or more terms?
- or is the option to exclude terms, author or keywords
- or is the option to not include child terms
- it would require the option "Hidden if empty" and all the content of the block would be hidden (to include a title for example)
- possibly an option for a specific post-template to display the first article or the first n (with a larger image for example, while remaining in the same query)
- and also, more advanced, other parameters of our dear WP_Query: post_parent, several post-types

**Media Texte**: But why isn't there a top or bottom option in addition to right or left?

**Post excerpt:** possibly be able to adjust the maximum number of characters


### Where is the taxonomy loop (like the Query loop)?
Am I the only one who needs it? Nobody uses the WP_Term_Query?

Small thing: Where did the image size option go on the image block when you are editing the site?
Am I the only one who needs it?

**A filter block** for the "archives" or the results page for example
- date or period
- tags, terms
- author


**The famous Navigation block:** well things are progressing but in use it is not really manageable, we must continue to see the menus decentralized from the publisher site as they were in the past.
How can I tell my novice users that it's easy?

Why is there almost no responsive support for blocks?
For the spacer and the columns, to begin with, it would be very useful.


It would be handy if you could apply multiple styles to one block, for example "My Outline Style", "My Round Style" and "My FX Style"
Because the combination of each of the possible variations becomes exponential.

It would be useful if we could give the block group a name that would help organize our content (like anchors do in list view but only in the editor); For example "area under navigation", "recent news area", "call to action area"...

### We need missing block which is by default so perennial which is very important:

- Accordion (just to be able to open and close content with a button)
- Tabs (the derivative)
- Icons
- Animation
- (ask Munir Kamal in the Gutenberg Blocks Library & Toolkit – Editor Plus)

### loop taxonomy
Read post metas and format them
An elementary slider / carousel (but more complete than that of JetPack)
Share on social networks (from WordPress wouldn't it be obvious that this is by default?)

Etc...



### Priority ?

Fonts API, Openverse, Multi-Styles are great features.
However, Multi-lingual should not be the first step since it affects, among other things, the template system?

**Shouldn't we properly finalize the site editor before anything else?**


**Next time I will tell you about the extensions which should be by default so that we can say: it's complete!**

I volunteer to help contribute to WordPress.

I'm not demanding, I just want the best for WordPress!

**Long live WordPress.**

Thanks and good luck.

Stephan

**Reply by hanneslsm** [reply]
This contains a lot of info and it must have taken quite a while to write this all up!

It would be going beyond the scope of this thread to discuss all the issues that have been raised.
I cannot speak for the team but what I can say is that a lot of issues you brought up are already in progress, discussed and tested. There are some that will not be implemented the way users would expect it (e.g. responsive controls, see intrinsic design) and I think this is great.

The main goal of opening the discussion was to raise awareness on prioritising more basic UX flaws and improve the experience this way. Your formulation "it's incomplete" sums it up exactly. An example is one issue I recently created: https://github.com/WordPress/gutenberg/issues/50502. It makes Gutenberg feel buggy. There must be a way found to tackle and solve these kind of issues much faster.
Thanks for supporting this discussion with you observations!

**Comment by hanneslsm** [comment]
Wordpress 6.3 is on its way to its release and in the last month I followed very closely the work on this project, created issues, got to know some contributors, got involved. 

I am extremely impressed by the community and what was achieved in the last months. Patterns, the Interactivity API, the Playground are setting the stage for what's to come. The distant future is looking pretty bright. But the here and now? It's a grim, and that's why I posting again.

On a side note, I am currently building a design system for Figma and Wordpress and theme (like Ollie), so that building beautiful themes becomes even easier, faster and good design becomes accessible for more people. 
However, sometimes I doubt if getting to know Gutenberg so well is all worth it or if I shouldn't go and sell websites built with Squarespace.

I am frustrated because as i wrote in the above post, the essential functions of the core blocks are still terrible.
I am sick of installing plugins for essential features that destroy the site perfomance, bloated themes, work arounds and hacks. I am not asking for responsive controls or fancy slider or so. I am talking about the very, very basic use of Gutenberg core blocks.

@mtias I'd like to address you directly because my hope is that you can steer the direction of the team. Can you please, please make sure we get the basics right? To be honest, I'm not just asking, I'm begging.
The visions you've laid out for phase 3 are amazing, and I am very much looking forward to such a Wordpress.
However, we're talking about real-time collaboration or admin interfaces, but what is it all worth if people don't know how to center a heading or can't change the hover color of button?
Please, let's focus on the UX of the core blocks and design tools. Let's get this done fast - we have seen how capable the community is. 

The editor is advertised to make content editing and theme building more accessible. If we do not wake up and finally fully focus on the essential functions and UX behaviour, WordPress will continue to lose trust, users, and we will loose the fight for the open Internet. 


<details><summary>By the way, here are what I mean with essential features </summary>
<p>
Today morning I created a page with dummy content and all blocks to test my theme. It might be a mix of feature requests and bugs, but they for sure all are things that ruin the user experience.
 This is what I came across, and this not even close to all I noticed:

- [ ] Table of content do not link automatically to the headings (even though the description says so)
- [ ] Table of content includes headings from the footer
- [ ] Site Logo: Is rendered blurry on the front end (not so in the editor)
- [ ] Navigation: Its not possible to change how the current page link looks like
- [ ] Paragraphs cannot have wide width
- [ ] Columns: Not possible to break out by using enter
- [ ] Cover Block: Inner blocks widths are ignored when content position is changed
- [ ] Cover Block: The width of groups that aligned at Bottom Center (or top or left or right) cannot have a filled width (The centered one does)
- [ ] Post Featured Image should be possible to transform to a cover block with post featured image as background
- [ ] File Block: Why is this an opinionated button but not the default one?
- [ ] Verse Block: not possible to exit via Enter
- [ ] Video Block: Handles are missing
- [ ] It’s still not possible to use svgs. C’mon - how long do we have this discussion already? Ten years?
- [ ] Buttons: It’s not possible to select hover colors
- [ ] Columns: It’s not possible to round corners for a column
- [ ] Indenting and outdenting list items still don't work using Tab
- [ ] Transforming a row block to columns is not as expected
- [ ] The search block cannot be styled correctly
- [ ] When I have text sizes defined, the cover block still uses a custom one
- [ ] it’s not possible to change the overlay of cover blocks to 95% (only 90% or 100%)
- [ ] It’s only possible to use line-heights of 1.2 or 1.3 but not 1.25
- [ ] It’s possible to add images from links but not so cover images
- [ ] It not possible to remove the link underline from post titles
- [ ] The gradient color picker is absolutely unfinished
- [ ] The table block. 
</p>
</details>

**Reply by annezazu** [reply]
I know this isn't addressed to me but I'd love to ensure these each have issues. Noting this for anyone following along if folks can help. For now, here's what I found quickly as preparation continues for 6.3. Of note, I know Matías has his own run ins with similar pain points: https://github.com/WordPress/gutenberg/issues/48443 I share this for context as I know these small details add up quickly and it's been a major focus to help resolve with project boards like [UX & Polish popping up](https://github.com/orgs/WordPress/projects/96/). More remains and feedback from folks like you help get us there. 

- Table of content do not link automatically to the headings (even though the description says so) & Table of content includes headings from the footer > I'd love if you could comment with what you found here: https://github.com/WordPress/gutenberg/issues/51605 
- Site Logo: Is rendered blurry on the front end (not so in the editor) > I can't replicate this with 6.3 RC1. Can you open an issue with what you're finding? 
- Navigation: Its not possible to change how the current page link looks like > https://github.com/WordPress/gutenberg/issues/42299
- Paragraphs cannot have wide width > https://github.com/WordPress/gutenberg/issues/31429
- Columns: Not possible to break out by using enter > opened this: https://github.com/WordPress/gutenberg/issues/52774
- Cover Block: Inner blocks widths are ignored when content position is changed > I am assuming this is related to https://github.com/WordPress/gutenberg/issues/37036 but please let me know if that's incorrect. 
- Cover Block: The width of groups that aligned at Bottom Center (or top or left or right) cannot have a filled width (The centered one does) > this seems to overlap with https://github.com/WordPress/gutenberg/issues/50610 but let me know if that's incorrect.
- Post Featured Image should be possible to transform to a cover block with post featured image as background > commented here: https://github.com/WordPress/gutenberg/issues/51750#issuecomment-1642568702
- Verse Block: not possible to exit via Enter > https://github.com/WordPress/gutenberg/issues/52773 
- Video Block: Handles are missing > I assume this relates to https://github.com/WordPress/gutenberg/issues/20944
- Buttons: It’s not possible to select hover colors > https://github.com/WordPress/gutenberg/issues/38277 
- Columns: It’s not possible to round corners for a column > https://github.com/WordPress/gutenberg/issues/41345 and added to UX & Polish
- Indenting and outdenting list items still don't work using Tab > https://github.com/WordPress/gutenberg/issues/45404
- When I have text sizes defined, the cover block still uses a custom one > can you open an issue for this? 
- it’s not possible to change the overlay of cover blocks to 95% (only 90% or 100%) > https://github.com/WordPress/gutenberg/issues/51625 thanks for opening this previously ❤️ 
- It’s only possible to use line-heights of 1.2 or 1.3 but not 1.25 > https://github.com/WordPress/gutenberg/issues/42998
- It not possible to remove the link underline from post titles > https://github.com/WordPress/gutenberg/issues/46180

For these, if you can provide more information, that would be super helpful:

>Transforming a row block to columns is not as expected
>The search block cannot be styled correctly
>The table block.
>The gradient color picker is absolutely unfinished
> File Block: Why is this an opinionated button but not the default one?
>It’s possible to add images from links but not so cover images

I realize I'm asking for more from you so thank you and please know I understand if you do not have more time and I'll do my best to follow up where I can!

**Reply by hanneslsm** [reply]
Thanks @annezazu - this is way more than I have expected! Thank you also creating some new issues.
As you correctly pointed out, a lot of the named issues exist already. 
The list was a random collection of things that just came up when working with the editor for an hour or two.
Some of the issues were created months and some even years ago. And this is my exactly my point: they totally could have been fixed already. I know that sometimes the underlying code has to be changed and things take time. I also know that resources are limited, especially on such an open source project. However, we need a better way to prioritise especially those issues that ruin the UX.

Later the week I will provide missing information and create some tickets. However, to put it provocatively (sorry), what's the point if some of them will not get touched until next year or 2025?

The UX & Polish Board was a good start in the right direction, but we still need focus on improvements for the core blocks and editor behaviours. If phase 3 starts soon, i'm not really excited about it, but rather worried that the UX will be neglected even more.

**Reply by hanneslsm** [reply]
<details><summary>As promised, here some more info. But please note, those were just random examples that everyone could have collected when working with the editor for a few minutes. My point is really that we have to change something in the project management and prioritisation of tickets. </summary>
<p>

- Transforming a row block to columns is not as expected → https://github.com/WordPress/gutenberg/issues/52857
- The search block cannot be styled correctly → https://github.com/WordPress/gutenberg/issues/51617 The search block is generally a good example of bad UX. Why are the settings for the toggling labels, change styles and so on in the toolbar?
- The table block. → Yes, the table block. 
- The gradient color picker is absolutely unfinished → https://github.com/WordPress/gutenberg/issues?q=is%3Aopen+is%3Aissue+author%3A%40me+Gradient
- File Block: Why is this an opinionated button but not the default one?  → https://github.com/WordPress/gutenberg/issues/52850
- It’s possible to add images from links but not so cover images → Images have to option "Insert from URL", Cover blocks don't. 

</p>
</details> 


Also, I spend the whole morning to change the hover color of a navigation and failed. Tried it with theme.json, custom block styles, custom css. Just to change the over color of a navigation. Am I expecting to much? The last 15 years I've build websites this would fit literally in just one line of css code. Now I cannot even do it after trying _for hours_. [Such a essential bug shouldn't be ignored for more then 6 months](https://github.com/WordPress/gutenberg/issues/47431). I mean, it's a navigation with links. Of course it should be possible to change the hover color of the links, or what am I missing here?
@m  I'm literally crying for help here. Please help. There needs to be done something.
It feels like the Wordpress team don't even want that block themes are getting build. And I don't know why I am even bother anymore… I just should have rated Gutenberg with 1 star as everyone else and gone my way. :(

**Reply by Ren2049** [reply]
For the right priorization of issues UX leadership would be needed. I think biggest dilemma is the "building for everyone" approach which makes focused UX work almost impossible because there's no defined audience to focus on.

I believe the "for everyone" approach is also the reason why there's no existing UX leadership, with the result that the develoment is reactionary based on who screams the loudest. I'm comfortable to say that because some issues just won't exist if there was a UX leadership. Designing for everyone is a good idea for WP as a whole, but not for a design tool like the block editor and if you're dependent of adoption by designers.

The block editor needs the adoption of designers because designers are needed to define the tasty parameters for automated pattern variations by AI. (that's where it's going for the industry as a whole), and with the command palette WP is also already on the way.

Normal users or marketers won't use the block editor, they will use mainly patterns and AI prompts generating patterns based on our design systems and tokens.

WP should hire a professional UX lead focusing on these different usecases. Block editor for savvy designers, WP admin and high level design tools (patterns, AI) for non designers, and good APIs for devs.

**Reply by annezazu** [reply]
>Later the week I will provide missing information and create some tickets. However, to put it provocatively (sorry), what's the point if some of them will not get touched until next year or 2025?

I hear you! I have nearly [200 issues opened](https://github.com/WordPress/gutenberg/issues/created_by/annezazu) and some I care about more than others. WordPress has been around for 20 years now and not everything gets addressed immediately with open source projects. Having an issue opened around a point of feedback allows it to be considered. No open issues, there's little to no chance for consideration. Anything you can do to contribute code wise would be awesome. Believe it or not folks are working full force on these efforts and, even then, there's still always more to do.

**Comment by hanneslsm** [comment]
I'm going to close this discussion.

Perhaps, I may have been a bit too intense in my previous comments. I got carried away in the moment, and if I came across as disrespectful, I apologize.

I want to acknowledge the excellent work done by everyone on WordPress who's involved in the project. I'll continue to contribute to the best of my ability.

To those who read this and felt a connection with my previous post - I encourage you to join in and contribute, so we can make WordPress even better, together.

**Comment by stevendmaddox** [comment]
A Solution / Work around...
Put the Video Block inSIDE of a Column Block... and resize the Column Block.
...worked for me, untill they fix it.

**Comment by bradandhaze** [comment]
> I know, the title is a little clickbaity, but hear me out.
> 
> Let me start by saying that this is not another typical 1-star review calling for a return to the classic editor. In fact, I wholeheartedly believe in the vision behind Gutenberg and the power of open source. As a web designer, I've been following the development of Gutenberg closely since years, from the official updates to GitHub issues and PRs. I admire the team's hard work and dedication to making it better. I'm particularly impressed by how @Thelmachido handles negative reviews in the support forum with grace, @jameskoster's stunning mockups, @jasmussen's deep knowledge of the whole WordPress system, @annezazu's friendly nature that keeps the whole crew together or @richtabor, who seems to prioritise a holistic and easy-to-use product, just to name a few I regularly see here. There are also many more invaluable contributions from other essential team members.
> 
> However, I can't deny the deep frustration I feel. I love WordPress and don't want to leave it behind, but the current state of Gutenberg's user experience is driving me to consider alternatives like Webflow, Wix, and Shopify, which I never thought I'd do. It feels like I'm losing the battle for an open web, and it goes against my conviction.
> 
> Recently, I had a client who needed an online shop. I struggled with choosing WooCommerce & Gutenberg because Gutenberg still feels immature. Reluctantly, I had to propose using Shopify, which felt like admitting defeat.
> 
> Last year, I decided to take the leap and quit my job to become a full-time web designer. Throughout this journey, I've been eagerly waiting for Gutenberg to reach its full potential. I've been patient, I've been supportive, and I've been hopeful. But, unfortunately, I've also been frustrated. You see, as much as I love Gutenberg and the idea of open source, I've had to rely on apps like Elementor to make sure my clients can easily navigate their websites. I've struggled with the idea of recommending Gutenberg to others, fearing it could harm my reputation as a web designer. Despite countless updates, I still see unresolved issues that have been around for years. It's difficult for me to rely on several third-party plugins to extend WordPress and Gutenberg’s capabilities like Elementor, only because Gutenberg still has so many unnecessary flaws. The different UX behaviours that the plugins introduce, security risks, and performance issues only raise more questions (and costs) for my clients. Many plugins have their justification and i love the concept that Wordpress is extensible through plugins. But when it comes to basic features that need to be filled with plugins, it just doesn't feel good.
> 
> My frustration grows when I try to contribute and raise issues with concrete examples of improvements, only to have my suggestions dismissed, and tickets closed because of duplicates that somehow have already disappeared into oblivion. These weren’t fancy special edge cases, but very basic features. I can't help but feel ignored and disregarded, and it's exhausting. There is so much going on in the development of Gutenberg, and I love it! However, I think it's crucial to focus at least for a few weeks on very basic UX improvements.
> 
> Let me give me some examples of what I mean: adding basic dimension controls for the video block (#50018), addressing gallery block issues (#50022 and #50021), or improving the UX when changing preferences (#50017). These low-hanging fruits can significantly improve the user experience without losing sight of Gutenberg's long-term goals. I recently was in a client's call and I was seriously ashamed of how often I had to say “this cannot be currently done, I'll install a plugin" or "you have to do it like this…" only to look at confused faces afterwards.
> 
> If WordPress 6.3 doesn't address these basic UX issues, I fear I'll have to move away from it entirely, and I don't know when or if my trust can be restored. It's a painful thought, especially since I love this community, believe in the mission of WordPress like democratising publishing (which doesn’t feel like when you have to invest a lot of time to get basic things working or invest money for plugins) and in the power of open source.
> 
> I wish I could get more involved in development, but I have to look after my own business. I would even be open to helping (if there is a sponsor out there, let me know) - at least until the very basic features are fixed. However, if the whole thing when trying to contribute is so frustrating, I don't even know if it's worth it.
> 
> I really think that creating a user-friendly and accessible Gutenberg experience should be prioritised more. Basic UX issues should be addressed to ensure that the mission and values of WordPress remain intact and continue to drive the platform's success. I know that you have heard this a lot already. But if this does not change, I see the fight for a free internet as lost. This is what frustrates me the most.

I'm new to it all but I just built a 853 page website before the summer and just got back form camping in France and ALL right re-usable blocks just disappeared from every page....that is every call to action - no idea why.  Beyone frustrating. https://www.legaldocuments.co.uk   Any help, signposting from anyone here would be very appreciated.

**Reply by hanneslsm** [reply]
This must be frustrating indeed! 

My original discussion post was rather leaning towards the prioritizing of features / bugs in this open source project and the user experience design of the editor. 
For your issue, it's hard to say what caused the error. A good place to ask for help are the forums where a lot of contributors will support you: https://wordpress.org/support/forums/

**Reply by annezazu** [reply]
So sorry to hear that :/ Noting that it might be that they are now called "synced patterns" per this update: https://wordpress.org/news/2023/07/synced-patterns-the-evolution-of-reusable-blocks/ Sharing in case that helps you identify what's going on. I'm also on the WordPress 6.4 release squad and if you can replicate/identify what's going on, please share in a new issue: https://github.com/WordPress/gutenberg/issues and feel free to @ me (@annezazu) so I can track down someone to help solve it. As part of the WordPress 6.3 squad though, I can tell you that we haven't had widespread reports nor have I seen any reports of them entirely disappearing from a site!
